src/main/react-front-end/src/views/termsComponent.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import terms from '../assets/terms.txt';
+
+export default class TermsComponent extends Component {
+    state = {
+        termsData: ''
+    }
+
+    readTextFile = file => {
+        var rawFile = new XMLHttpRequest();
+        rawFile.open("GET", file, false);
+        rawFile.onreadystatechange = () => {
+            if (rawFile.readyState === 4) {
+                if (rawFile.status === 200 || rawFile.status == 0) {
+                    var allText = rawFile.responseText;
+                    console.log("allText: ", allText);
```

Comment:
This log can be removed or commented out

Reply:
done